### PR TITLE
[R2] Add upload/download examples to rclone example

### PR DESCRIPTION
**Pull request by KianNH**
Changed provider to `Cloudflare` which is supported in 1.59 - 1.59 is now in stable

### CODE REVIEW

**Comment by KianNH** [comment]
https://patch-62.cloudflare-docs-github.pages.dev/r2/examples/rclone/

**Comment by KianNH** [comment]
FWIW, I didn't test the 5TB scenario due to not, uh, having 5TB of disk space.

image

It'll happily accept 10GiB (over the max 5GiB size of a single `PutObject`) call since it uses multipart uploads though - so I see 0 reason why that wouldn't work.

File: content/r2/examples/rclone.md [hunk 1]
```diff
@@ -60,7 +64,25 @@ $ rclone tree r2demo:my-bucket-name
 # └── todos.txt
 ```
 
-You can also generate presigned links which allow you to share public access to a file temporarily.
+## Uploading & retrieving objects
```
Comment (deadlypants1973) [inline content/r2/examples/rclone.md]:
```suggestion
## Upload and retrieving objects
```

File: content/r2/examples/rclone.md [hunk 2]
```diff
@@ -45,6 +45,10 @@ acl = private
 
 You may then use the new `rclone` provider for any of your normal workflows.
 
+## Listing buckets & objects
```
Comment (deadlypants1973) [inline content/r2/examples/rclone.md]:
```suggestion
## List buckets & objects
```

File: content/r2/examples/rclone.md [hunk 3]
```diff
@@ -60,7 +64,25 @@ $ rclone tree r2demo:my-bucket-name
...
+$ rclone tree r2demo:user-uploads
+# /
+# ├── foobar.png
+# └── dog.txt
+
+# Download dog.txt from the user-uploads bucket
+$ rclone copy r2demo:user-uploads/dog.txt dog.txt
+```
+
+## Generating presigned URLs
```
Comment (deadlypants1973) [inline content/r2/examples/rclone.md]:
```suggestion
## Generate presigned URLs
```

File: content/r2/examples/rclone.md [hunk 4]
```diff
@@ -60,7 +64,25 @@ $ rclone tree r2demo:my-bucket-name
 # └── todos.txt
 ```
 
-You can also generate presigned links which allow you to share public access to a file temporarily.
+## Uploading & retrieving objects
+
+The [rclone copy](https://rclone.org/commands/rclone_copy/) command can be used to upload objects to an R2 bucket and vice versa - this allows you to upload files up to the 5TB maximum object size that R2 supports.
```
Comment (deadlypants1973) [inline content/r2/examples/rclone.md]:
```suggestion
The [rclone copy](https://rclone.org/commands/rclone_copy/) command can be used to upload objects to an R2 bucket and vice versa - this allows you to upload files up to the 5 TB maximum object size that R2 supports.
```

File: content/r2/examples/rclone.md [hunk 5]
```diff
@@ -64,7 +64,7 @@ $ rclone tree r2demo:my-bucket-name
 # └── todos.txt
 ```
 
-## Uploading & retrieving objects
+## Upload and retrieving objects
```
Comment (deadlypants1973) [inline content/r2/examples/rclone.md]:
```suggestion
## Upload and retrieve objects
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**